_playbook/automated-testing.md

```diff
@@ -28,23 +47,19 @@ The Axe accessibility engine is now built into [Google's Lighthouse](https://dev
 
 Government Digital Services in both the [United Kingdom](https://github.com/alphagov/accessibility-monitoring) and [Singapore](https://github.com/GovTechSG/purple-hats) have leveraged Deque's open-source axe-core accessibility engine to create tools that crawl and gather accessibility barriers for both public-facing websites and internal web-based communications. Similar open-source accessibility engines have been built by IBM & Deque, and there are proprietary tools available from firms like WebAim, Tenon & Siteimprove.
 
-However, it is important to note that, no matter the tool, [automated testing only catches about one third of accessibility issues](https://alphagov.github.io/accessibility-tool-audit/), so it cannot be relied upon to determine if a site complies. To better ensure that their digital tools are accessible to keyboard-only and assistive technology users, automated tests need to be done along with a robust manual testing plan.
+### Automated testing caveat 
```

Comment:
@dmundra what do you think about changing this subhead to something like “Popular tools and a caveat”? I think it describes what follows a little better.

Reply:
@dmundra never mind, I was misreading the diff file. I think it’s great as-is.